x-pack/plugins/security_solution/public/overview/pages/overview.tsx

```diff
@@ -91,20 +82,22 @@ const OverviewComponent = () => {
             <SiemSearchBar id="global" indexPattern={indexPattern} />
           </FiltersGlobal>
 
-          <StyledSecuritySolutionPageWrapper>
+          <SecuritySolutionPageWrapper>
             {!dismissMessage && !metadataIndexExists && canAccessFleet && (
               <>
                 <EndpointNotice onDismiss={dismissEndpointNotice} />
                 <EuiSpacer size="l" />
               </>
             )}
-            <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween">
-              <SidebarFlexItem grow={false}>
-                <StatefulSidebar />
-              </SidebarFlexItem>
-
-              <EuiFlexItem grow={true}>
-                <EuiFlexGroup direction="column" gutterSize="none">
+            <EuiFlexGroup>
+              <EuiShowFor sizes={['xl']}>
+                <EuiFlexItem grow={1}>
+                  <StatefulSidebar />
+                </EuiFlexItem>
+              </EuiShowFor>
```

Comment:
I don't think this column is critical to the page. This removes it from mobile views.